M2/Macaulay2/tests/normal/ringmap7.m2

```diff
@@ -0,0 +1,21 @@
+restart
```

Comment:
I think the restart here is going to prevent this test from actually running.

Reply:
Argh!  I put that in to test the additional asserts I added but forgot to remove it.  Thanks!